suite-common/graph/src/types.ts

```diff
@@ -35,3 +49,34 @@ export type GroupedBalanceMovementEvent = {
     date: Date;
     payload: GroupedBalanceMovementEventPayload;
 };
+
+export type AccountHistoryBalancePoint = {
+    time: number;
+    cryptoBalance: string;
+};
+
+export type AccountBalanceHistoryWithTokens = {
+    main: AccountHistoryBalancePoint[];
+    tokens: Record<TokenAddress, AccountHistoryBalancePoint[]>;
+};
+
+export type AccountHistoryMovementItem = {
+    time: number;
+    txs: number;
+    received: BigNumber;
+    sent: BigNumber;
+    sentToSelf: BigNumber;
+};
+
+export type AccountHistoryMovement = {
```

Comment:
nit:
```suggestion
export type AccountMovementHistory = {
```
Isn't it better naming? Since the object describes `history` rather than `historical movement`.

Reply:
It describes historical movements in specific block so I think it's fine, but I don't have any strong opinion about it.

Reply:
If it describes movements only in a single block and not across the whole history, than it is fine I guess 👍.